src/server/plugins/router.ts

```diff
@@ -64,19 +65,23 @@ export default {
         method: 'post',
         path: '/help/cookie-preferences',
         handler(request, h) {
-          const decision = request.payload['cookies[additional]']
+          const decision = request.payload['cookies[additional]'].toLowerCase()
 
-          if (decision === 'yes') {
-            request.yar.set('cookieConsent', true)
-          } else if (decision === 'no') {
-            request.yar.set('cookieConsent', false)
-          } else {
+          if (!['yes', 'no'].includes(decision)) {
             throw Boom.badRequest('Unknown cookie preference')
           }
 
+          h.state('cookie_consent', decision)
           request.yar.flash('cookieConsentUpdated', true, true)
 
           return h.redirect(request.info.referrer)
+        },
+        options: {
+          validate: {
+            payload: Joi.object({
+              'cookies[additional]': Joi.string().valid('yes', 'no').required()
```

Comment:
Can you rename `additional` to `analytics` and same with the content?

We've got a mixture of two Design System examples at the moment

**Additional cookies**
>We’d like to set additional cookies so we can remember your settings, understand how people use the service and make improvements.

**Analytics cookies**
>We’d also like to use analytics cookies so we can understand how you use the service and make improvements.